model/image.go

```diff
@@ -134,3 +134,17 @@ type MultipartUploadMsg struct {
 	// reader pointing to the beginning of the artifact data
 	ArtifactReader io.Reader
 }
+
+// MultipartGenerateImageMsg is a structure with fields extracted from the multipart/form-data
+// form sent in the artifact generation request
+type MultipartGenerateImageMsg struct {
+	Name                  string    `json:"name"`
+	Description           string    `json:"description"`
```

Comment:
May add an additional `omitempty` json tag here, same goes for other optional parameters (if any).

Reply:
I actually want all the fields to be posted as parameters to the workflow, so I won't add `omitempty` in this specific case.